lib/ansible/modules/include_vars.py

```diff
@@ -70,6 +70,12 @@
     type: bool
     default: no
     version_added: "2.7"
+  merge_hashes:
+    description:
+      - Merges existing hash variables instead of overwriting them.
```

Comment:
I would add note (this falls back to the global hash_behaviour configuration).

Reply:
Done. Please feel free to change the wording (or the name of the parameter) as you see fit.